pkg/daemon/ceph/osd/daemon_test.go

```diff
@@ -213,6 +213,7 @@ NAME="sdb1" SIZE="30" TYPE="part" PKNAME="sdb"`, nil
 		{Name: "sdd", DevLinks: "/dev/disk/by-id/scsi-cdef /dev/disk/by-path/pci-c:d:e:f-scsi-1", RealPath: "/dev/sdd"},
 		{Name: "sde", DevLinks: "/dev/disk/by-id/sde-0x0000 /dev/disk/by-path/pci-0000:00:18.0-ata-1", RealPath: "/dev/sde"},
 		{Name: "nvme01", DevLinks: "/dev/disk/by-id/nvme-0246 /dev/disk/by-path/pci-0:2:4:6-nvme-1", RealPath: "/dev/nvme01"},
+		{Name: "nvme02", DevLinks: "/dev/disk/by-id/nvme-0247 /dev/disk/by-path/pci-0:2:4:6-nvme-2", RealPath: "/dev/nvme02"},
```

Comment:
This change fails some of the assertions below that checks the size of `mapping.Entries`

Reply:
just remove it.